models/pod.go

```diff
@@ -22,6 +23,7 @@ type Pod struct {
 	Status              string            `json:"status"`
 	AppLabel            bool              `json:"appLabel"`
 	VersionLabel        bool              `json:"versionLabel"`
+	CustomDashboards    []string          `json:"-"` // not used in the UI but needed to build app and workload models
```

Comment:
Minor: If it's harmless I'm +1 to expose it into the json, perhaps it's useful for debugging purposes to fetch the workload's pods and check if things are ok.

Reply:
ok i'll do it